GUI/Main.Designer.cs

```diff
@@ -1431,9 +1433,15 @@ private void InitializeComponent()
             this.reportAnIssueToolStripMenuItem.Size = new System.Drawing.Size(230, 30);
             this.reportAnIssueToolStripMenuItem.Text = "Report an issue...";
             this.reportAnIssueToolStripMenuItem.Click += new System.EventHandler(this.reportAnIssueToolStripMenuItem_Click);
-            // 
+            //
+            // compatibleKSPVersionsToolStripMenuItem
+            //
+            this.compatibleKSPVersionsToolStripMenuItem.Name = "compatibleKSPVersionsToolStripMenuItem";
+            this.compatibleKSPVersionsToolStripMenuItem.Size = new System.Drawing.Size(233, 24);
+            this.compatibleKSPVersionsToolStripMenuItem.Text = "Compatible KSP versions";
+            //
```

Comment:
`this.kSPInstanceSettingsToolStripMenuItem.Click += new System.EventHandler(this.CompatibleKspVersionsToolStripMenuItem_Click);`

I think you missed this line